WIC/Constants/WICPixelFormats.cs

```diff
@@ -0,0 +1,98 @@
+﻿using System;
+
+namespace WIC.Constants
+{
+    public static class WICPixelFormats
+    {
+        public static readonly Guid GUID_WICPixelFormatDontCare = new Guid(0x6fddc324, 0x4e03, 0x4bfe, 0xb1, 0x85, 0x3d, 0x77, 0x76, 0x8d, 0xc9, 0x00);
```

Comment:
Could you please remove the `GUID_` name prefix from this and all following fields?

(I know that's what the documentation and original header files call it, but there's no added benefit in this prefix; I've taken some liberties with renaming things elsewhere to get a little closer to .NET's usual conventions.)